src/main/target/DALRCF405/target.h

```diff
@@ -48,12 +53,20 @@
 #define USE_ACC
 #define GYRO_1_CS_PIN           PA4
 #define GYRO_1_SPI_INSTANCE     SPI1
-#define GYRO_1_ALIGN            CW90_DEG
-#define ACC_1_ALIGN             CW90_DEG
 
+#if defined(VIVAF4AIO)
+//------ICM20602
+# define USE_GYRO_SPI_ICM20602
+# define USE_ACC_SPI_ICM20602
+# define GYRO_1_ALIGN            CW0_DEG
+# define ACC_1_ALIGN             CW0_DEG
```

Comment:
`ACC_1_ALIGN` is deprecated and will be removed in #8076. Probably best to remove both occurrences to avoid conflict depending on the merge order.